charts/kong/values.yaml

```diff
@@ -397,6 +397,13 @@ livenessProbe:
   successThreshold: 1
   failureThreshold: 3
 
+# Proxy container lifecycle hooks
+# Ref: https://kubernetes.io/docs/concepts/containers/container-lifecycle-hooks/
+lifecycle:
+  preStop:
+    exec:
+      command: ["/bin/sh", "-c", "sleep 30 && kong quit"]
```

Comment:
Default value of `terminationGracePeriodSeconds` is 30 seconds and `kong quit` can itself take some time.
Can we change this to be 15 or 20 seconds?

Reply:
oops, I accidentally committed the sleep, I meant to keep the value identical to the chart's current value. I just updated it to 15 seconds if you do want to keep that in there.

Reply:
I think it is OK to sleep for 15 seconds so that other controllers/LBs have enough time to stop sending requests to Kong. 

@rainest @artburkart Thoughts?

Reply:
I think making it configurable is the biggest win here. If the default value proves to be too much or too little, the user of the chart is empowered to configure it according to their needs.

That said, 15 seconds seems like a reasonable amount to me and including it as part of the chart serves as a cue to users that it's something they may want to tweak. 

I'll need to test it to provide more substantial feedback. Odds are, I'll try it out this week. Thanks for surfacing this!